redirects/14.0.txt

```diff
@@ -42,8 +42,18 @@ applications/general/calendars/google/google_calendar_credentials.rst applicatio
 
 # applications/inventory
 
-applications/inventory_and_mrp/inventory/routes/concepts/inter_warehouse.rst applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
+applications/inventory_and_mrp/inventory/management/delivery/dropshipping.rst applications/inventory_and_mrp/inventory/shipping/operation/dropshipping.rst                                                  # /inventory/management/incoming/* --> /shipping/operation/*
```

Comment:
to update here in odoo/odoo in 16.0
https://github.com/odoo/odoo/blob/47333b5b13823ac2812a120f2cb443cb68d7d92c/addons/purchase_stock/views/res_config_settings_views.xml#L20

Reply:
The comment are not correct but it doesn't matter that much ^^ 
- you wrote /inventory/ in all of them, implying that it's removed 
- on this line you wrote incoming instead of delivery

```suggestion
applications/inventory_and_mrp/inventory/management/delivery/dropshipping.rst applications/inventory_and_mrp/inventory/shipping/operation/dropshipping.rst                                                  # /management/delivery/* --> /shipping/operation/*
```